Html bem

Pwarde opened the pull request:
Please review my HTML/BEM setup,

Pwarde commented:
Build success! 👍

raymonschouwenaar commented:
# Assignment 1

## General
- ARIA looks good! But i’m missing here and there some tabindex, roles and aria-lables
- Check here how you can [review the accessibility](https://developers.google.com/web/fundamentals/accessibility/how-to-review)
- Make sure to open Devtools and open the Accessibility tab to check all Aria attributes. To check if everything is good, go to the Audits tab and run the test with only the option for Accessibility.

## BEM
- Most of your classes are good! But the “section” is not a real clear name. I would name the section with all the bars, skills matrix or something. Just think about the layout of a portfolio website for someone.

Pwarde commented:
@raymonschouwenaar 
Pushed new version. The changes only include:
- Added tab indexes
- Added a closingtag to the TextArea element that was forgotten.

As per [Google accessibility fundamentals](https://developers.google.com/web/fundamentals/accessibility/how-to-review#try_it_with_a_screen_reader), roles and aria labels only need to be added to control elements.

Site reads fine with ChromeVox.

The use of the section class (in BEM) is for the general section styling. In SCSS could be done with a mixin.
